packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -201,7 +202,9 @@ export interface StreamProps {
    * If you choose KMS, you can specify a KMS key via `encryptionKey`. If
    * encryption key is not specified, a key will automatically be created.
    *
-   * @default - StreamEncryption.KMS if encryptionKey is specified, or StreamEncryption.UNENCRYPTED otherwise
+   * @default - StreamEncryption.KMS if encrypted Streams are supported in the region
+   * or StreamEncryption.UNENCRYPTED otherwise.
```

Comment:
These should be indented I think:

```ts
   * @default - StreamEncryption.KMS if encrypted Streams are supported in the region,
   *   or StreamEncryption.UNENCRYPTED otherwise;
   *   StreamEncryption.KMS if an encryption key is supplied through the encryptionKey property
   */
```

Reply:
done